src/routes/+layout.svelte

```diff
@@ -53,4 +54,6 @@
 </script>
 
 <slot />
-<NotificationsList />
+<NotificationList class="fixed top-0 right-0 mt-[54px] p-6 space-y-4 overflow-y-auto max-h-[calc(100vh-54px)]" let:notification position={Position.TopRight}>
```

Comment:
Why use both?

- `position={Position.TopRight}`
- `fixed top-0 right-0`

Reply:
Using `position={Position.TopRight}` doesn't work for some reason